build.gradle

```diff
@@ -961,6 +963,54 @@ project(':streams') {
   }
 }
 
+project(':streams:streams-scala') {
+  println "Building project 'streams-scala' with Scala version ${versions.scala}"
+  apply plugin: 'scala'
+  archivesBaseName = "kafka-streams-streams-scala"
+
+  // use experimental flag to enable SAM type conversions in compiler
+  if (versions.baseScala == '2.11') {
+    tasks.withType(ScalaCompile) {
+      scalaCompileOptions.additionalParameters += ["-Xexperimental"]
+    }
+  }
+
+  dependencies {
+    compile project(':streams')
+
+    compile libs.scalaLibrary
+
+    testCompile project(':core')
+    testCompile libs.avro4s
+    testCompile libs.scalaLogging
+    testCompile libs.logback
```

Comment:
We don't use logback for anything else. I'd suggest keeping it consistent with the project.

Reply:
done ..